src/platform/nrfconnect/wifi/WiFiManager.h

```diff
@@ -168,6 +168,13 @@ class WiFiManager
     static constexpr uint16_t kRouterSolicitationIntervalMs        = 4000;
     static constexpr uint16_t kMaxInitialRouterSolicitationDelayMs = 1000;
     static constexpr uint8_t kRouterSolicitationMaxCount           = 3;
+    static constexpr uint32_t kConnectionRecoveryMinIntervalMs     = CONFIG_CHIP_WIFI_CONNECTION_RECOVERY_MINIMUM_INTERVAL;
+    static constexpr uint32_t kConnectionRecoveryMaxIntervalMs     = CONFIG_CHIP_WIFI_CONNECTION_RECOVERY_MAXIMUM_INTERVAL;
+    static constexpr uint32_t kConnectionRecoveryJitterMs          = CONFIG_CHIP_WIFI_CONNECTION_RECOVERY_JITTER_INTERVAL;
+    static constexpr uint32_t kConnectionRecoveryDelayToReset      = CONFIG_CHIP_WIFI_CONNECTION_RECOVERY_RESET_DELAY;
+
+    static_assert(kConnectionRecoveryMinIntervalMs < kConnectionRecoveryMaxIntervalMs);
```

Comment:
Maybe this can be assured even earlier, i.e. by adding some conditions/limits in Kconfig definitions, I' not sure. @kkasperczyk-no?